openlibrary/plugins/openlibrary/lists.py

```diff
@@ -28,6 +29,83 @@ def GET(self):
         return render_template("lists/home")
 
 
+class lists_partials(delegate.page):
+    path = "/lists/partials"
+
+    def GET(self):
+        i = web.input(key=None)
+
+        user = get_current_user()
+        doc = self.get_doc(i.key)
+        seed_info = self.get_seed_info(doc)
+        user_lists = self.get_user_lists(user.get_lists(sort=True), seed_info)
+
+        dropper = render_template('lists/dropper_lists', user_lists)
+        active = render_template(
+            'lists/active_lists', user_lists, user['key'], seed_info
+        )
+
+        partials = {
+            'dropper': str(dropper),
+            'active': str(active),
+        }
+
+        return delegate.RawText(json.dumps(partials), content_type="application/json")
+
+    def get_doc(self, key):
+        if key.startswith("/subjects/"):
+            return subjects.get_subject(key)
+        return web.ctx.site.get(key)
+
+    def get_seed_info(self, doc):
+        """Takes a thiing, determines what type it is, and returns a seed summary"""
+        if doc.key.startswith("/subjects/"):
+            seed = doc.key.split("/")[-1]
+            if seed.split(":")[0] not in ["place", "person", "time"]:
+                seed = "subject:" + seed
```

Comment:
```suggestion
                seed = f"subject:{seed}"
```
Smaller and faster... https://www.scivision.dev/python-f-string-speed